Oryx.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2042
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30011.22
 MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
nit: I think for now it's better if we keep min vs version as 16.0.* as well

Reply:
So this is OK right?

Reply:
I would still like to change the minimum version to 16